flink-connectors/flink-connector-kafka-base/src/main/java/org/apache/flink/streaming/util/serialization/JsonRowDeserializationSchema.java

```diff
@@ -37,6 +38,7 @@
  *
  * <p>Failure during deserialization are forwarded as wrapped IOExceptions.
  */
+@Internal
```

Comment:
This could also be `@PublicEvolving`, I don't think there's a problem with the user using it directly.